### ISSUE TITLE
Re-publish bootstrap-growl

### ISSUE THREAD

**Issue by achingbrain** [mouse0270/bootstrap-notify#102]
Looks like you unpublished the `bootstrap-growl` module from npm when you renamed this project from `bootstrap-growl` to `bootstrap-notify`.

Could you please republish it under that name?  You've broken every single module out there that depends on `bootstrap-growl`.

**Comment by mouse0270** [comment]
@pgilad Would you be willing to loog into this? 

I am sadly not very familiar with npm because I dont use it.

**Comment by pgilad** [comment]
Hey @achingbrain what do you mean unpublished?
I just installed it `bootstrap-growl` using `npm install bootstrap-growl` into a clean directory and it installed fine. I also went ahead and installed `bootstrap-notify` alongside it to show the diff:
![image](https://cloud.githubusercontent.com/assets/4533329/8763444/82eeab02-2da4-11e5-9dd8-b724f2c89952.png)

**Comment by pgilad** [comment]
@achingbrain Also I tried installing `guvnor` using npm and it installed fine. What exactly is the problem?

**Comment by mouse0270** [comment]
@pgilad Thank you very much for looking into this. I will have to eventually learn npm. There are just so many things out there and I haven't had time to learn the ones I don't have to use for work. I am grateful that you looked into this.

**Comment by achingbrain** [comment]
@pgilad have you run `npm install bootstrap-growl` before?  If so npm probably installed it from your local cache.  Try removing it first:

```sh
$ rm -rf ~/.npm/bootstrap-growl
$ npm install bootstrap-growl
npm ERR! fetch failed https://registry.npmjs.org/bootstrap-growl/-/bootstrap-growl-2.0.0.tgz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
```

**Comment by pgilad** [comment]
@achingbrain You're absolutely right. @mouse0270  Can you add me as a collaborater to `bootstrap-growl` on npm? I can try to republish the new version.

**Comment by achingbrain** [comment]
I think that since it's been unpublished, the name is now free for anyone to use on npm so you shouldn't need to be added, you can just publish.

Bear in mind this though, from [the docs](https://docs.npmjs.com/cli/unpublish):

> Even if a package version is unpublished, that specific name and version combination can never be reused. In order to publish the package again, a new version number must be used.

**Comment by pgilad** [comment]
@achingbrain I couldn't republish version 2.0.0.
I created a standalone npm module called [bootstrap-growl-legacy(https://www.npmjs.com/package/bootstrap-growl-legacy) which has versions 2.0.0 and 2.0.1 as posted on `bower`.

Hopefully you guys can switch smoothly to that module, and meanwhile start thinking about upgrading to version 3.1.1 and use `bootstrap-notify`.

Sorry for the trouble..
